### DISCUSSION TITLE
What is the templater command to display previous quarter?

### DISCUSSION THREAD

**Discussion by rasivasu** [SilentVoid13/Templater#1151] [Help] [answered]
Hi,

Currently, I am using the following command but this is prone to error when the template is translated during the last week of a quarter.  Is there a better way to go about doing this?

`<% tp.date.now("YYYY-[Q]Q", -90) %>`

The above command will output the following when run on 25 July 2023.

`2023-Q2`

Thanks.

**Comment by Zachatoo** [comment, marked as answer]
This should work consistently

```js
<% moment().subtract(1, 'Q').format("YYYY-[Q]Q") %>
```

**Comment by rasivasu** [comment]
Thank you very much, Zach

On Wed, Jul 26, 2023 at 7:11 AM Zach Young ***@***.***> wrote:

> This should work consistently
>
> <% moment().subtract(1, 'Q').format("YYYY-[Q]Q") %>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SilentVoid13/Templater/discussions/1151#discussioncomment-6546083>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIZULQBYHKVYR3LXTE4O4DXSBYV3ANCNFSM6AAAAAA2XLWWWQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards,
Ravi
